config/default.yml

```
@@ -198,6 +198,11 @@ Gemspec/RequiredRubyVersion:
 
 #################### Layout ###########################
 
+Gemspec/RubyVersionGlobalsUsage:
+  Description: Checks usage of RUBY_VERSION in gemspec
+  Enabled: true
+  VersionAdded: '0.72'
```

```suggestion
  VersionAdded: '0.72'
  Include:
    - '**/*.gemspec'
```

Can you add `Include` to execute the cop only on gemspec files?